### PR TITLE
botmeta: vultr migrated to ngine_io.vultr

### PR DESCRIPTION
**Pull request by resmo**
##### SUMMARY
vultr related ansible content has been migrated to dedicated collection. Also see https://github.com/ansible-collections/community.general/pull/172

##### ISSUE TYPE

- Docs Pull Request

##### COMPONENT NAME

vultr
##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: .github/BOTMETA.yml [hunk 1]
```diff
@@ -794,7 +794,7 @@ files:
     - cloud
     - vultr
     maintainers: $team_vultr
-    migrated_to: community.general
+    migrated_to: ngine_io.vultr
```
Comment (bcoca) [inline .github/BOTMETA.yml:797]: you also need to update the per file references

**Review by bcoca — DISMISSED** [review]

**Comment by resmo** [comment]
ready_for_review

**Comment by resmo** [comment]
ready_for_review

**Review by felixfontein — APPROVED** [review]

**Comment by gundalow** [comment]
rebuild_merge

**Pull request merged**